packages/react/src/components/List/ListContent/ListContent.jsx

```diff
@@ -257,10 +262,24 @@ const ListContent = ({
       ...(!hasChildren && item.hasLoadMore ? [renderLoadMore(item, isLoadingMore, level)] : []),
     ];
   };
+  const previousExpandedIds = usePrevious(expandedIds, expandedIds);
+  const notifyOnLastExpansionChange = (itemId) => {
```

Comment:
Hmm... there is a problem if we wanna support "toggle" and not only "onExpand" changes since this does not work if you expand row 1, then expand row 2 and then collapse row 1.

Reply:
But perhaps we can take the last item from the diff between expandedIds and previousExpandedIds?

Reply:
Yep! The diff works. I have to do it both ways (ie. the diff of current to previous and then previous to current) to handle both opens and closes, but it seems to work regardless of order.